tools/src/main/java/io/github/ygojson/tools/dataprovider/impl/yugipedia/mapper/MarkupStringMapper.java

```diff
@@ -43,6 +45,18 @@ public MarkupString mapToJapanese(final String property) {
 	 * @return property without markup.
 	 */
 	public String map(final MarkupString property) {
+		if (property == null) {
+			return null;
+		}
 		return property.withoutMarkup();
 	}
+
+	public List<MarkupString> mapToNewLineList(final String property) {
+		return map(property).splitByNewLine(true).toList();
```

Comment:
Add some tests to the `MarkupStringMapper` new methods to check for the null values. This will throw a NPE and we want to prevent it, so also adapt

Reply:
This doesn't throw a NPE because the MarkupString container accept null values and handle them propertly. Anyway, tests were implemented.